Lib/platform.py

```diff
@@ -10,7 +10,8 @@
 """
 #    This module is maintained by Marc-Andre Lemburg <mal@egenix.com>.
 #    If you find problems, please submit bug reports/patches via the
-#    Python bug tracker (http://bugs.python.org) and assign them to "lemburg".
+#    Python bug tracker (https://github.com/python/cpython/issues) and
```

Comment:
```suggestion
#    Python bug tracker (https://devguide.python.org/tracker/) and
```

Reply:
This was marked as solved, but the suggestion hasn't been committed.  Not sure if it was intentional or an oversight.

Reply:
For the target audience here, I don't think the devguide adds much. The main info is who to tag.